source/compatibility.tex

```diff
@@ -64,6 +64,28 @@
                                 // changed according to the global C locale
 \end{codeblock}
 
+\rSec2[diff.cpp23.depr]{\ref{depr}: compatibility features}
+
+\nodiffref
+\change
+Remove the type alias \tcode{allocator<T>::is_always_equal}.
+\rationale
+Non-empty allocator classes derived from \tcode{allocator} needed to explicitly
+define an \tcode{is_always_equal} member type so that \tcode{allocator_traits}
+would not use the one from the allocator base class.
+\effect
+It is simpler to correctly define an allocator class with an allocator base
+class. For example:
+\begin{codeblock}
+template <class T>
+struct MyAlloc : allocator<T> {
+  int tag;
+};
+
+static_assert(!allocator_traits<MyAlloc<int>>::is_always_equal);  // Error in \CppXXIII{}, OK in \CppXXVI{}
```

Comment:
The comment should be aligned to a column whose number is a multiple of 8.
Current column number: 66
Try adding six spaces.